sqlglot/dialects/bigquery.py

```diff
@@ -962,6 +1003,7 @@ class Generator(generator.Generator):
             exp.Values: _derived_table_values_to_unnest,
             exp.VariancePop: rename_func("VAR_POP"),
             exp.SafeDivide: rename_func("SAFE_DIVIDE"),
+            exp.Export: _export_sql,
```

Comment:
This will be unnecessary if you move the generation logic within the base generator class.
```suggestion
```